staging/src/k8s.io/code-generator/cmd/conversion-gen/generators/conversion.go

```diff
@@ -179,6 +179,10 @@ func getManualConversionFunctions(context *generator.Context, pkg *types.Package
 			}
 			manualMap[key] = f
 		} else {
+			// prevent user error when they don't get the correct conversion signature
+			if strings.HasPrefix(f.Name.Name, "Convert_") {
+				glog.Errorf("Rename function %s %s -> %s to match expected conversion signature", f.Name.Package, f.Name.Name, buffer.String())
+			}
```

Comment:
what's the goal here? Drop the commit? Or merge both outputs?

Reply:
nit: put the message below into the else branch.

Reply:
Him - I definitely wanted to be able to get a consistent error in the debug scenario (i.e., if you've  jacked logging up to 8, I want to filter on "wrong name" via grep).  So it was quasi intentional to display both at v8.

Reply:
Ok, no strong opinion here. Am ok with how it is.